hyperjumptech/monika
# Fix: Remove Pako as it is no longer needed

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
This PR fixes #1043 

## How did you implement / how did you fix it  
Remove Pako as it was the root cause of the gzipping the report.

## How to test  
- Run Symon in local
- Run this app https://github.com/nicnocquee/monika-demo (should be running on port 3001 because Symon occupies port 3000)
- Reverse proxy the app using ngrok by running ngrok http 3001
- Get the URL (should be like this http://03e1-180-252-170-36.ngrok-free.app/)
- Add the URL http://03e1-180-252-170-36.ngrok-free.app/api/demo/status to the Symon probe page
- Wait for it to report the result

## Screenshot
![Screenshot_20230512_161216](https://github.com/hyperjumptech/monika/assets/16670475/24d69fc0-3985-4672-b942-03478df2a838)
